src/main/resources/templates/error.html

```diff
@@ -10,17 +10,29 @@
 <body class="overflow-y-visible bg-white">
   <div layout:fragment="content" class="m-4">
 
-    <h1 class="font-inter font-medium text-32 mb-4">
-      <span th:text="${errorStatus} ?: 'N/A'"></span>
+    <h1 class="font-inter font-medium text-32 text-red mt-4">
+      Error
     </h1>
 
-    <div class="flex items-center mt-4 bg-pink border-2 border-red">
-      <i class="fas fa-exclamation-circle text-red ml-2"></i>
-      <p class="font-inter font-medium text-16 text-red ml-2" th:text="${errorMessage}"></p>
-    </div>
+    <table
+      class="attribute-list w-full text-14 text-black font-medium font-inter border-separate border-spacing-0 border-grey-200 border-2 rounded-xl overflow-hidden mt-4">
+      <tbody>
+        <tr>
+          <th
+            class="p-4 text-white font-semibold bg-grey-300 text-left align-text-top whitespace-nowrap w-0 min-w-[200px]">
+            Status Code</th>
+          <td class="p-4 border-grey-200 border-l-1 text-left align-text-top" th:text="${errorStatus} ?: 'N/A'"></td>
+        </tr>
+        <tr>
+          <th
+            class="p-4 text-white font-semibold bg-grey-300 text-left align-text-top whitespace-nowrap border-grey-200 border-t-1">
+            Message</th>
+          <td class="p-4 border-grey-200 border-l-1 text-left align-text-top border-t-1" th:text="${errorMessage}"></td>
+        </tr>
+      </tbody>
+    </table>
```

Comment:
Make use of `attribute-list.ts` to generate this instead. You can see `case-report.ts` for an example of that. Some notes:

* Add the model attributes as data properties on the div container for the attribute list. Then you can read these into a typescript object
* You'll need to add a new script for the error page, which must be configured as an entry point in `webpack.config.js`. Create the attribute definitions and then build the attribute list on page load

This solution is a bit more complex, but keeps the project more maintainable as all attribute lists are handled in the same way by the same code.